tests/unit/health/mapping/test_es.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+# from health.mapping import es
+
+from tests.unit import test
+
+
+class InitElasticTestCase(test.TestCase):
```

Comment:
This should go like a separated commit (because it's change of unit test that is not related to changes in main)